tests/test_cmd_enc_gainmap_boxes_golden.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# Same as test_cmd_enc_boxes_golden.sh but for when gain maps are enabled.
```

Comment:
This .sh does not exist anymore

Reply:
It does still exist, it's just much shorter now since most of the content has been moved to golden_test_common.sh

Reply:
So much read, I thought it was deleted :)